features/raadpleeg-verblijfplaats-met-periode/gba/test/opschorting-bijhouding-gba.feature

```diff
@@ -0,0 +1,208 @@
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde              |
+      | type                | RaadpleegMetPeriode |
+      | burgerservicenummer | 000000012           |
+      | datumVan            | 2022-01-01          |
+      | datumTot            | 2023-01-01          |
+      Dan heeft de response een object met de volgende gegevens
+      | naam     | waarde                                                         |
+      | type     | https://datatracker.ietf.org/doc/html/rfc7231#section-6.5.4    |
+      | title    | Opgevraagde resource bestaat niet.                             |
+      | status   | 404                                                            |
+      | detail   | De persoon met burgerservicenummer 000000012 is niet gevonden. |
+      | code     | notFound                                                       |
+      | instance | /haalcentraal/api/brphistorie/verblijfplaatshistorie           |
+
+  Rule: bij het raadplegen van een persoon met afgevoerde persoonslijst wordt alleen de reden en datum opschorting geleverd
+    Overige gegevens op de afgevoerde persoonslijst worden niet geleverd
+
+    Abstract Scenario: historie wordt gevraagd van persoon waarvoor de bijhouding is opgeschort met reden "F" (fout)
+      Gegeven de persoon met burgerservicenummer '000000012' is ingeschreven op adres 'A1' met de volgende gegevens
+      | gemeente van inschrijving (09.10) | datum aanvang adreshouding (10.30) |
+      | 0800                              | 20220801                           |
+      En de persoon heeft de volgende 'inschrijving' gegevens
+      | naam                                 | waarde   |
+      | reden opschorting bijhouding (67.20) | F        |
+      | datum opschorting bijhouding (67.10) | 20220829 |
+      Als gba verblijfplaatshistorie wordt gezocht met de volgende parameters
+      | naam                | waarde              |
+      | type                | RaadpleegMetPeriode |
+      | burgerservicenummer | 000000012           |
+      | datumVan            | <datum van>         |
+      | datumTot            | <datum tot>         |
+      Dan heeft de response 0 verblijfplaatsen
+      En heeft de response de volgende gegevens
+      | naam                                     | waarde   |
+      | opschortingBijhouding.reden.code         | F        |
+      | opschortingBijhouding.reden.omschrijving | fout     |
+      | opschortingBijhouding.datum              | 20220829 |
+
+      Voorbeelden:
+      | omschrijving                   | datum van  | datum tot  |
+      | voor datum opschorting         | 2022-08-01 | 2022-08-11 |
+      | na datum opschorting           | 2022-09-01 | 2022-10-01 |
+      | overlapt met datum opschorting | 2022-08-01 | 2022-10-01 |
+
+  Rule: opschortingBijhouding wordt geleverd
+
+    Abstract Scenario: historie wordt gevraagd van persoon waarvoor de bijhouding is opgeschort
```

Comment:
Ik snap deze scenario niet.
Kan iemand mij toelichten hoe dit speelt  ten op zichte van de rule

Rule: bij het raadplegen van een persoon met afgevoerde persoonslijst wordt alleen de reden en datum opschorting geleverd
    Overige gegevens op de afgevoerde persoonslijst worden niet geleverd

Wat is afgevoerde (Is dat alleen F? Is dat meer dan F?) en waarom zal bij de gehighlighte scenario wel verblijfplaatsen terug gegeven?